### ISSUE TITLE
dossier personnalisé

### ISSUE THREAD

**Issue by ArthurLamiral** [nextcloud/android#5512]
### Steps to reproduce
1. Auto upload
2. Dossier personnalisé
3. Dossier source

### Expected behaviour
- Tell us what should happen
Sélection du dossier source
### Actual behaviour
- Tell us what happens
Aucune sélection mémorisée
### Environment data
Android version:
9
Device model: 
Ulefone Armor x5
Stock or customized system:

Nextcloud app version:
3.10
Nextcloud server version:
18.1
#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
Hi @ArthurLamiral,

canb you post a screenshot, do you see any error message?

**Comment by ArthurLamiral** [comment]
Bonjour,

ci-jointe une capture écran de la page qui refuse d'enregistrer la configuration source et destination des dossiers à sauvegarder.

Le serveur Nextcloud utilisé fonctionne très bien avec un PC et synchronise plusieurs répertoires du PC.

Le problème est uniquement avec Android.

Cordialement.

Arthur
![_20200224_140414](https://user-images.githubusercontent.com/61087128/75155888-41e7eb00-5711-11ea-9fba-4f0d6122b90f.png)

**Comment by AndyScherzinger** [comment]
You might have to click several times on the item (unfortunately).

This is an issue -don't know why the action isn't triggered continuously all the time :( just yet

**Comment by AndyScherzinger** [comment]
cc @tobiasKaminsky @ezaquarii @ArisuOngaku for input ❤️

**Comment by ezaquarii** [comment]
I'm not sure I follow:

> You might have to click several times on the item (unfortunately).

@AndyScherzinger do you know what the problem is?

**Comment by AndyScherzinger** [comment]
Unfortunately no, it seems that the click event on the text works sometimes while sometimes it just doesn't (for the folder items) while it seems to always work for the line items with the check boxes. So I am a bit confused how that could happen...

**Comment by ArisuOngaku** [comment]
@AndyScherzinger @ezaquarii Tested this (happens on latest master too); let me clarify what the bug is: when you click on the folder rows, it always works *except* if you click on the second text line (either "Choose {...} local folder..." or the actual chosen path). So it looks like the click event is interrupted by that element.

@ArthurLamiral parvenez-vous à ouvrir la boite de dialogue de sélection du dossier ? (je veux clarifier la nature du problème que vous souhaitez rapporter)

**Comment by joshtrichards** [comment]
Fixed by #5593.
